charts/fybrik/templates/fybrik-admin-crb.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ template "fybrik.fullname" . }}-admin-crb
+  name: {{ .Release.Namespace }}-{{ template "fybrik.fullname" . }}-admin-crb
```

Comment:
Can a limit on the name length be exceeded?

Reply:
Since we only create the clusterrolebinding when running cluster scoped = true. We could remove the namespace from the metadata.name if concerned about long namespace names.